packages/x-charts/src/BarChart/BarElement.tsx

```diff
@@ -117,12 +124,28 @@ export function BarElement(props: BarElementProps) {
   };
   const classes = useUtilityClasses(ownerState);
 
+  const spring = useSpring({
```

Comment:
Do you know if our competitors are bundling a third party animation library like this?

Reply:
- Nivo is using spring
- Rechart is using react-transition-group -> sounds limited for animating props modification
- Echarts has its own set of function -> mAybe to much work for now

Reply:
OK
One advantage of `react-transition-group` is that we already bundle it for the pickers so most users will have it in their bundle anyway.
But if it's too limited, I agree that going for `react-spring` can be interesting.

Reply:
FWIW in my last projects I've always been using this [minimalistic animation function](https://gist.github.com/romgrk/0627faac9c6f810c28e0fc56efefd9a8), I was very satisfied with the choice because it's very simple to understand & tune. Adding a wrapper around it to make it into a hook should be trivial.

Reply:
I spend some time to think about it, but will go with react-spring for now, more for their `animated`  elements than for the interpolation functions. It allows to pass style as usual without having react re-render

https://www.react-spring.dev/docs/concepts/animated-elements